BookPlayer/Library/LibraryViewController.swift

```diff
@@ -58,15 +60,27 @@ class LibraryViewController: BaseListViewController, UIGestureRecognizerDelegate
         UIApplication.shared.endReceivingRemoteControlEvents()
     }
 
+    fileprivate func toggleEmptyStateView() {
+        guard let placeholder = self.emptyStatePlaceholder else {
+            return
+        }
+
+        if self.items.isEmpty {
```

Comment:
Any particular reason as to why not just hide it instead of removing it from the superview?

Reply:
The view isn't in the superview (see Storyboard), so I'd have to add it once in the viewDidLoad and I figured it would be cleaner that way, but if you think otherwise, we can change it back.

Reply:
🤔 but it *is* in the superview, for clarification's sake, I don't mean the `BaseListViewController`. What worries me about removing it from the superview instead of hiding it, is that it might turn into nil after removing it, since the superview is the one that holds the strong reference to that variable

Reply:
Hmm true.